### DISCUSSION TITLE
💱 Currency Switch Apps/Plugin Feature Add in Fluxstore Shopify

### DISCUSSION THREAD

**Discussion by ecomdeveloperbd** [inspireui/support#5639] [New Feature/Ideas Requests]
Currency Switch Apps/Plugin Feature Add in Fluxstore Shopify 

it's most important for Shopify store, Because Shopify store mostly used Dropshipping. Dropshipping User multiple country, so currency Switch plugin/app integration.

**Comment by osman-kocak** [comment]
+1
